services/frontend/src/components/AnalysisVisualization.vue

```diff
@@ -1,20 +1,136 @@
 <template>
-  <div>
-    <!-- create graphs here -->
-    <!-- this is a mock -->
-    <img src="mock_files/mock_analysis_graph.png"/>
+  <div v-if="analisStared" class="sector flex-column-items-centered card"> Start Analysis before starting </div>
```

Comment:
"analysis" non "analis". Probably this is not an isolated incident. I suggest quick Find&Replace, e.g. in vscode you can do it on all files in workspace at once.